cypress.json

```diff
@@ -1 +1,4 @@
-{ "defaultCommandTimeout": 8000 }
+{
+  "baseUrl": "https://qa-react-conduit.herokuapp.com/",
```

Comment:
This should probably be in the .env

Reply:
This is on the Cypress documentation, it's ok!